modules/tensor_mechanics/include/materials/ADRadialReturnCreepStressUpdateBase.h

```diff
@@ -9,39 +9,4 @@
 
 #pragma once
 
-#include "RadialReturnStressUpdate.h"
-
-/**
- * This class provides baseline functionallity for creep models based on the stress update material
- * in a radial return isotropic creep calculations.
- */
-class ADRadialReturnCreepStressUpdateBase : public ADRadialReturnStressUpdate
-{
-public:
-  static InputParameters validParams();
-
-  ADRadialReturnCreepStressUpdateBase(const InputParameters & parameters);
-
-  /**
-   * Compute the strain energy rate density for this inelastic model for the current step.
-   * @param stress The stress tensor at the end of the step
-   * @param strain_rate The strain rate at the end of the step
-   * @return The computed strain energy rate density
-   */
-  virtual ADReal
-  computeStrainEnergyRateDensity(const ADMaterialProperty<RankTwoTensor> & /*stress*/,
-                                 const ADMaterialProperty<RankTwoTensor> & /*strain_rate*/)
-  {
-    mooseError(
-        "The computation of strain energy rate density needs to be implemented by a child class");
-  }
-
-protected:
-  virtual void initQpStatefulProperties() override;
-  virtual void propagateQpStatefulProperties() override;
-  virtual void computeStressFinalize(const ADRankTwoTensor & plastic_strain_increment) override;
-
-  /// Creep strain material property
-  ADMaterialProperty<RankTwoTensor> & _creep_strain;
-  const MaterialProperty<RankTwoTensor> & _creep_strain_old;
-};
+#include "RadialReturnStressUpdateBase.h"
```

Comment:
Is it as simple as this @loganharbour ?

Reply:
Crap...I misspelled...but you get the idea...

Reply:
Yep - I think that'll do it, as long as the class that existed there before is now defined in your new header.